index.html

```diff
@@ -93,160 +94,700 @@ <h2> Powered by creators everywhere. </h2>
     <main>
         <div class="flex justify-evenly">
             <div class="flex column gallery-column">
+                <div class="gallery-column_image relative">
 
-                <img srcset="./assets/Picture-1_small.jpg 400w,
+                    <div class="overlay"> <a class="image-card__profile" href="https://unsplash.com/@hiestudio"> Zhao
+                            Yangyang </a>
+                    </div>
+
+                    <img srcset="./assets/Picture-1_small.jpg 400w,
                              ./assets/Picture-1_medium.jpg 800w, 
                              ./assets/Picture-1_large.jpg 1500w" alt="picture-1" sizes="
                              (min-width: 980px) 31vw,
                              (min-width: 700px) and (max-width: 980px) 45vw,
                              100vw" src="./assets/Picture-1_medium.jpg" loading="lazy" />
-                <img srcset="./assets/Picture-2_small.jpeg 400w, 
+
+                    <div class="overlay flex justify-between mx-2">
```

Comment:
This shouldn't have the `mx-2` class. Because of this, the overlay is outside the image.